src/components/parameters/ParameterRecStruct.svelte

```diff
@@ -36,7 +36,7 @@
         name: key,
         order: index,
         schema: schema.items[key],
-        value: value ? value[key] || null : null,
+        value: value !== null ? value[key] : null,
```

Comment:
🥷

```ts
value[key] ?? null
```

Reply:
Won't this throw an error if value is null and you try to access a property on it? Unless we're sure this will never be null.

Reply:
Ah yeah! You're right. Nothing to see here.